package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "waterdataui-assets",
+  "version": "1.0.0",
+  "description": "Compiled static assets for Water Data For The Nation",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "clean": "rm -rf dist && mkdir dist",
+    "build": "npm run clean && run-p build:*",
+    "build:css": "node-sass --include-path node_modules/uswds/src/stylesheets assets/styles/main.scss | uglifycss > dist/main.css",
+    "build:fonts": "mkdir -p dist/fonts && cp node_modules/uswds/src/fonts/* dist/fonts",
+    "build:images": "mkdir -p dist/img && cp -r node_modules/uswds/src/img/* dist/img",
+    "build:js": "browserify -t uglifyify assets/scripts/index.js | uglifyjs -c > dist/bundle.js",
+    "serve:static": "live-server --port=9000 --cors --no-browser dist",
+    "watch": "run-p watch:* serve:static",
+    "watch:css": "nodemon -w assets/styles -e scss -x \"npm run build:css\"",
+    "watch:fonts": "nodemon -w node_modules/uswds/src/fonts -x \"npm run build:fonts\"",
+    "watch:images": "npm run build:images",
+    "watch:js": "watchify assets/scripts/index.js -o dist/bundle.js -dv",
+    "watch:python": "./run.py"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/usgs/waterdataui.git"
+  },
+  "author": "USGS",
+  "license": "CC0-1.0",
+  "bugs": {
+    "url": "https://github.com/usgs/waterdataui/issues"
+  },
+  "homepage": "https://github.com/usgs/waterdataui#readme",
+  "dependencies": {
+    "babel-core": "^6.26.0",
+    "babel-preset-env": "^1.6.1",
+    "babelify": "^8.0.0",
+    "browserify": "^14.5.0",
+    "live-server": "^1.2.0",
+    "node-sass": "^4.7.2",
+    "nodemon": "^1.13.3",
+    "npm-run-all": "^4.1.2",
+    "rollupify": "^0.4.0",
+    "uglify-js": "^3.2.2",
+    "uglifycss": "0.0.27",
+    "uglifyify": "^4.0.5",
+    "uswds": "^1.4.4",
+    "watchify": "^3.9.0"
+  },
+  "browserify": {
+    "transform": [
+      "rollupify",
```

Comment:
If we use ES6 imports, `rollupify` can "tree shake" unused code out of the JS bundle. Unfortunately, the CommonJS modules that the 18F code uses can't benefit from this.

Reply:
possible future pull request on the standards repo?